src/app/container/launch/launch.component.html

```diff
@@ -15,45 +15,53 @@
   -->
 
 <div *ngIf="_selectedVersion">
-  <span>
-    <div class="form-inline">
-      <label tooltip="Commands to launch workflow from the command line">Launch With</label>
-      <div class="pull-right">
-        <select id="launchWithDescriptor" class="form-control input-sm" [(ngModel)]="currentDescriptor" (change)="reactToDescriptor()">
-          <option *ngFor="let descriptor of validDescriptors" [ngValue]="descriptor.value">
-            {{ descriptor.value }}
-          </option>
-        </select>
-      </div>
+  <div *ngIf="!_selectedVersion.valid">
+    <label tooltip="Commands to launch tool from the command line">Launch With</label>
+    <div class="alert alert-warning" role="alert">
+      This section is only available for valid versions.
```

Comment:
For this alert, we should probably say what the user can do to fix the situation (i.e. provide valid descriptors).
For workflows at least, still undecided about tools

Reply:
Undecided, maybe we explain and vote in stand-up

Reply:
Ok so this is fine to merge with just the workflow change, and we can add the tool comment at a later time?